src/controllers/entity.controller.ts

```diff
@@ -52,6 +52,10 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
       :  {};
     data[IsOnRegisterInCountryFormedInKey] = (data[IsOnRegisterInCountryFormedInKey]) ? +data[IsOnRegisterInCountryFormedInKey] : '';
 
+    // Wipe 'register in country formed in' data if IsOnRegisterInCountryFormedInKey is no or not selected
+    data[PublicRegisterNameKey] = (data[IsOnRegisterInCountryFormedInKey]) ? data[PublicRegisterNameKey] : '';
```

Comment:
Why checking the field `data[IsOnRegisterInCountryFormedInKey] `twice?

Reply:
i can tidy that up, was just following pattern of that function